build_tools/github/check_wheels.py

```diff
@@ -8,14 +8,8 @@
 with gh_wheel_path.open("r") as f:
     wheel_config = yaml.safe_load(f)
 
-build_matrix = wheel_config["jobs"]["build_wheels"]["strategy"]["matrix"]
-n_python_versions = len(build_matrix["python"])
-
-# For each python version we have: 7 wheels
-# 1 osx wheel (x86_64)
-# 4 linux wheel (i686 + x86_64) * (manylinux1 + manylinux2010)
-# 2 windows wheel (win32 + wind_amd64)
-n_wheels = 7 * n_python_versions
```

Comment:
While we are at it, could you please improve the error message when `n_dist_files != n_wheels` to print the observed list of `dist_files` below?